src/frontend/apps/e2e/__tests__/app-impress/common.ts

```diff
@@ -40,7 +40,8 @@ export const createDoc = async (
       })
       .click();
 
-    const input = page.getByRole('textbox', { name: 'doc title input' });
+    const input = page.getByLabel('doc title input');
+    await expect(input).toHaveText('');
```

Comment:
why test this ? can be better to test if the input is visible

Reply:
When you create a new doc from a doc, the value does not have the time to be clear cos the provider takes a few milliseconds to clear. To add a check that the value is empty, it adds a bit of times for the provider to be clean.

Reply:
all right